### ISSUE TITLE
Cheese Factory doesn't produce cheese

### ISSUE THREAD

**Issue by LNXlady** [ashjack/Sim-U-Kraft-Reloaded#56]
The worker collects milk and it says they are doing various cheesy chores like stirring the cheese, slicing the cheese, etc., but they never actually make or sell any cheese.

**Comment by AwesomeDonut67** [comment]
What version of simukraft are you using?

**Comment by LNXlady** [comment]
AwesomeDonut67, I no longer have it installed to check but it was the latest version available at the time.

**Comment by teagan75** [comment]
Yes my cheese maker makes no cheese and my glassmaker puts no glass in chests after cooking it. I am on version 1.0.4a

**Comment by xXXIMMATTHEWXXx** [comment]
Hello! I am porting this port to try and update the inactivity from the person who updated this mod. Please [recreate the issue here.](https://github.com/xXXIMMATTHEWXXx/Sim-u-kraft-Restored)
